packages/gafl-webapp-service/src/pages/multibuy/view-licences/route.js

```diff
@@ -38,6 +39,17 @@ export const getData = async request => {
     data.uri.add_licence = addLanguageCodeToUri(request, ADD_LICENCE.uri)
   }
 
+  data.licences.forEach(licence => {
+    const licenceDetails = [
+      { key: { text: mssgs.licence_summary_licence_holder }, value: { text: licence.licenceHolder } },
+      { key: { text: mssgs.licence_summary_type }, value: { text: licence.type } },
+      { key: { text: mssgs.licence_summary_length }, value: { text: licence.length } },
+      { key: { text: mssgs.licence_summary_start }, value: { text: licence.start } },
+      { key: { text: mssgs.licence_summary_price }, value: { text: mssgs.pound + licence.price } }
+    ]
+    data.licenceInfo.push(licenceDetails)
+  })
```

Comment:
Thanks for moving this here. I would say, it's slightly preferable to use `Array.map` for transformations rather than a `forEach` loop because it's slightly more succinct, but it's pretty minor so don't worry too much. I'll probably mutter about it if I ever have to modify this file and change it then 😜